include/ipc/ipc_service.h

```diff
@@ -157,21 +223,190 @@ int ipc_service_register_endpoint(const struct device *instance,
 
 /** @brief Send data using given IPC endpoint.
  *
- *  @param ept Registered endpoint by @ref ipc_service_register_endpoint.
- *  @param data Pointer to the buffer to send.
- *  @param len Number of bytes to send.
+ *  @param[in] ept Registered endpoint by @ref ipc_service_register_endpoint.
+ *  @param[in] data Pointer to the buffer to send.
+ *  @param[in] len Number of bytes to send.
  *
  *  @retval -EIO when no backend is registered or send hook is missing from
- *	    backend.
+ *               backend.
  *  @retval -EINVAL when instance or endpoint is invalid.
- *  @retval -EBADMSG when the message is invalid.
+ *  @retval -EBADMSG when the data is invalid (i.e. invalid data format,
+ *		     invalid length, ...)
  *  @retval -EBUSY when the instance is busy.
  *
  *  @retval bytes number of bytes sent.
  *  @retval other errno codes depending on the implementation of the backend.
  */
 int ipc_service_send(struct ipc_ept *ept, const void *data, size_t len);
 
+/** @brief Get the TX buffer size
+ *
+ *  Get the maximal size of a buffer which can be obtained by @ref
+ *  ipc_service_get_tx_buffer
+ *
+ *  @param[in] ept Registered endpoint by @ref ipc_service_register_endpoint.
+ *
+ *  @retval -EIO when no backend is registered or send hook is missing from
+ *		 backend.
+ *  @retval -EINVAL when instance or endpoint is invalid.
+ *  @retval -ENOTSUP when the operation or the timeout is not supported by backend.
```

Comment:
timeout is probably leftover. Could yo remove it?

Reply:
damn